### DISCUSSION TITLE
exploring-dotnet-6-part-5-supporting-ef-core-tools-with-webapplicationbuilder/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#16] [Announcements]
# exploring-dotnet-6-part-5-supporting-ef-core-tools-with-webapplicationbuilder/

In this post I describe how HostBuilder had to be updated to support the EF Core tools after the introduction of WebApplication and WebApplicationBuilder

https://andrewlock.net/exploring-dotnet-6-part-5-supporting-ef-core-tools-with-webapplicationbuilder/

[a comment by an automated account is not shown]

**Comment by JoaoCampos07** [comment]
Super interesting, in the end i also feel that there is some abuse on "DiagnosticSource " :D. Very creative solution eitherway.
